### ISSUE TITLE
Gift Card Payment Method Displayed Briefly on Payment Page

### ISSUE THREAD

**Issue by amit-parmar-7** [Adyen/adyen-salesforce-commerce-cloud#1084]
**Describe the bug**
On the Payment Page, Observe that the 'Gift card' Payment method is displayed for the Split Second and then disappears and all other payment methods are displaying.

**To Reproduce**

1.Go to the storefront as a Registered or Guest User
2.Add Product to the cart.
3.Proceed to checkout
4.After Filling in the Details for the Shipping page, Click on the “Next: Payment“ button.
5.Observe the Payment Page.

**Expected behavior**
On the payment page, the gift card component should not load if it is not enabled.

**Screenshots**
![image-20240516-125104](https://github.com/Adyen/adyen-salesforce-commerce-cloud/assets/143161224/87dba2ad-82b5-46ca-afae-b7b7bd8f7d6b)

**Desktop (please complete the following information):**
Chrome Desktop 125.0.6422.61 (Official Build) (64-bit)

**Comment by shanikantsingh** [comment]
Hi @amit-parmar-7 , what is the cartridge version and SFRA version?

**Comment by amit-parmar-7** [comment]
Hi @shanikantsingh, I'm using Adyen cartridge v24.1.0 and SFRA v7.0.0

**Comment by olga-adyen** [comment]
Hi @amit-parmar-7 
we tested the checkout on v24.1.0 and SFRA 7, when gift cards are not available in the customer area, the div is hidden for us. 
![image](https://github.com/Adyen/adyen-salesforce-commerce-cloud/assets/98472671/f517462c-fd12-4c8f-ac2f-0df188500eba)

We had this issue previously, but we fixed it in v23.2.1 with [this PR](https://github.com/Adyen/adyen-salesforce-commerce-cloud/pull/935).

I could suggest that you check that your implementation is not modified, for example by comparing your local project repo and the default v24.1.0 repo with a Diff tool, similar KDIFF or [MELD](https://meldmerge.org/).
